common/src/main/scala/com/gu/media/youtube/package.scala

```diff
@@ -56,6 +55,75 @@ package object youtube {
     }
   }
 
+  case class YouTubeMetadataUpdate(
+    title: Option[String],
+    categoryId: Option[String],
+    description: Option[String],
+    tags: List[String],
+    license: Option[String],
+    privacyStatus: Option[String]
+  ) {
+    def clean(): YouTubeMetadataUpdate = {
```

Comment:
Yeah, I wasn't overly pleased with it. I've updated to use method chaining which is slightly better.